### ISSUE TITLE
[MultiSelect] Popup not opening/closing when clicking right after the cursor

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#1219]
When clicking right after the cursor in order to select a new item, the Popup is not opened.

Check the following sample plunker demonstrating the issue:

[http://plnkr.co/edit/Zhy0OVHIAOgJa3ZwUe5S?p=preview](http://plnkr.co/edit/Zhy0OVHIAOgJa3ZwUe5S?p=preview)

**Comment by tapopov** [comment]
Seems related to https://github.com/telerik/kendo-angular/issues/1191

**Comment by gyoshev** [comment]
Workaround: bind to the click event and toggle the popup. 

```html
 <kendo-multiselect
    [data]="listItems"
    #multiselect
    (click)="!multiselect.isOpen && multiselect.toggle(true)">
 </kendo-multiselect>
```

[Plunker](https://plnkr.co/edit/fn237ef6HSi7p6P7i5if?p=preview).

**Comment by ipeshev** [comment]
Available in [latest dev version](https://github.com/telerik/kendo-angular-dropdowns/releases/tag/v3.0.3-dev.201808031552).

**Comment by tsvetomir** [comment]
Shipped in v3.0.3
